src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java

```diff
@@ -0,0 +1,85 @@
+package com.devotedmc.ExilePearl.command;
+
+import com.devotedmc.ExilePearl.ExilePearl;
+import com.devotedmc.ExilePearl.ExilePearlApi;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.SkullMeta;
+import vg.civcraft.mc.civmodcore.inventory.items.ItemUtils;
+import vg.civcraft.mc.civmodcore.inventorygui.DecorationStack;
+import vg.civcraft.mc.civmodcore.inventorygui.IClickable;
+import vg.civcraft.mc.civmodcore.inventorygui.MultiPageView;
+
+public class CmdShowAllPearls extends PearlCommand {
+
+	private static final DateFormat DATE_FORMAT = new SimpleDateFormat("dd MMM yyyy");
+
+	public CmdShowAllPearls(final ExilePearlApi pearlApi) {
+		super(pearlApi);
+		this.aliases.add("showall");
+		this.setHelpShort("Lists all pearled players.");
+
```

Comment:
if this is intentionally available to all players to use, a mirror to this that would be nice is some kind of "checkif" command.

But this looks good!

Reply:
Not sure what you mean

Reply:
some command to specifically check if a particular player, by name, is pearled.

/checkif Protonull

> Yes, they are pearled, here they are, etc.

Reply:
just an idle thought while looking this over, safe to ignore ;)

Reply:
Aaaaah, okay get it now, definitely sounds like a good idea. I'm not really sure how to do commands with parameters with this particular command system though o.o

Reply:
huh, turns out it exists, at least for admins:

https://github.com/CivClassic/ExilePearl/blob/master/src/main/java/com/devotedmc/ExilePearl/command/CmdAdminCheckExiled.java

can tell how often I used this... :D